submissions/SamVal007/Tiny-JS-OOP/index.js

```diff
@@ -0,0 +1,63 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: https://samval007.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Person {
+  constructor(species, gender, name, speach, legs) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.speach = speach;
+    this.legs = legs;
+  }
+
+  saySpeach() {
```

Comment:
How this method name translates?
Can you devise a better name for it?

Reply:
Yep! I found, that correct to write "speech" instead of "speach" :) 
but let's call this method just 'say()' - i think it's better.